### DISCUSSION TITLE
Map not displaying in Safari

### DISCUSSION THREAD

**Discussion by ChasBelov** [transitland/transitland#370] [Q&A] [answered]
If I go to https://www.transit.land/map, I can display the map in Google Chrome but not in Safari. In Safari, on the privacy tab, I am permitting cookies and have the other privacy options checked. I do not have a personal style sheet active.

In Safari, I get the navigation links, the routes/departures/options tab, and Select Routes. Zoom in to select routes. 

Just, no map. No map on the Departures tab either.

I am using macOS 12.5.1, Safari Version 15.6.1 (17613.3.9.1.16), and Chrome Version 104.0.5112.101 (Official Build) (x86_64).

<img width="1822" alt="Screen Shot 2022-08-21 at 2 40 29 PM" src="https://user-images.githubusercontent.com/59780179/185812154-d4603dd4-a60b-4444-8618-3a21ab06b714.png">

**Comment by drewda** [comment]
Hello, we can't reproduce this error. The map loads and works when I load it in the same version of Safari on the same version of macOS.

Please try clearing your cache and reloading.

If the problem persists, open the Safari developer tools and let us know what is in the console tab and the network tab as the page loads.

**Comment by ChasBelov** [comment, marked as answer]
Thank you. Still happening after clearing caches.

Console tab:

https://www.transit.land/_nuxt/7eed803.js
Error: Failed to initialize WebGL
(anonymous function) — 7eed803.js:27601:147
n — 7eed803.js:27173:232
initMap — 7eed803.js:343
initMap
mounted — 7eed803.js:281
ee — 1bedb00.js:621
vn — 1bedb00.js:1352
insert — 1bedb00.js:1103
T — 1bedb00.js:2860
(anonymous function) — 1bedb00.js:2944
(anonymous function) — 1bedb00.js:1858
r — 1bedb00.js:3267
(anonymous function) — 1bedb00.js:1435
(anonymous function) — 1bedb00.js:1481
Cn — 1bedb00.js:1379
(anonymous function) — 1bedb00.js:685
ce — 1bedb00.js:652
n - promiseReactionJob
f -(anonymous function)
- 7eed803.js:1807
If (anonymous function) - 7eed803.is:27601:120
n
- 7eed803.js:27173:232
initMap
•7eed803.js:343
N| initMap
§ mounted
- 7eed803.js:281
If ee
1bedb00.js:621
If vn
1bedb00.is:1352
f insert - 1bedb00.js:1103
T
- 1bedb00.is:2860
If (anonymous function) - 1bedb00.js:2944
If (anonymous function)
- 1bedb00.is:1858
-5r
- 1bedb00.js:3267
£ (anonymous function)
- 1bedb00.js:1435
If (anonymous function)
- 1bedb00.is:1481
If Cn - 1bedb00.js:1379
If (anonymous function) - 1bedb00.js:685
ce
1bedb00.js:652
N promise ReactionJob

https://www.transit.land/_nuxt/7eed803.js
Error: Failed to initialize WebGL.
n — 7eed803.js:27174
initMap — 7eed803.js:343
initMap
mounted — 7eed803.js:281
ee — 1bedb00.js:621
vn — 1bedb00.js:1352
insert — 1bedb00.js:1103
T — 1bedb00.js:2860
(anonymous function) — 1bedb00.js:2944
(anonymous function) — 1bedb00.js:1858
r — 1bedb00.js:3267
(anonymous function) — 1bedb00.js:1435
(anonymous function) — 1bedb00.js:1481
Cn — 1bedb00.js:1379
(anonymous function) — 1bedb00.js:685
ce — 1bedb00.js:652
n - promiseReactionJob
f - re
If ne
- 1bedb00.is:641
1bedb00.is:636
If te
1bedb00.js:613
fee
1bedb00.is:625
If vn
- 1bedb00.is:1352
If insert
1bedb00.is:1103
1bedb00.is:2860
f (anonymous function)
1bedb0C
f (anonymous function)
- 1bedb00
r
- 1bedb00.js:3267
F (anonymous function)
1bedb00
f (anonymous function)
- 1bedb00
If Cn
1bedb00.is:1379
If (anonymous function) - 1bedb00
F ce
1bedb00.is:652
N promiseReactionJob

Network tab:
[www.transit.land.har.cleared.txt](https://github.com/transitland/transitland/files/9430695/www.transit.land.har.cleared.txt)

**Reply by irees** [reply]
This looks like it could be an issue with WebGL support on your system and browser - can you try going to https://get.webgl.org/ and seeing if it can resolve the problem, or report further information back to us.
